website/content/en/docs/Getting Started/latest/schema.md

```diff
@@ -59,7 +59,7 @@ string
 <td>
 <code>metadata</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectmeta-v1-meta">
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta">
```

Comment:
The content for this file is generated by the command `make scheme-doc-gen` https://github.com/jenkinsci/kubernetes-operator/blob/master/Makefile#L395. The new content generation will overwrite your changes. To make it durable fix the https://github.com/jenkinsci/kubernetes-operator/blob/master/gen-crd-api-config.json config and apply changes in the golang comments https://github.com/jenkinsci/kubernetes-operator/tree/master/pkg/apis/jenkins/v1alpha2.  Then run `make scheme-doc-gen` and copy generated content to the `website/content/en/docs/Getting Started/latest/schema.md`.